File: CC/Codigos_Fonte_Relatorio_1/micro01/micro01.java

public class micro01{

    public static void main(String[] args){
        double cel, far;
        System.out.println("    Tabela de conversão: Celsius -> Fahrenheit");
        System.out.print("Digite a temperatura em Celsius: ");          
        cel = Double.parseDouble(System.console().readLine());
        far = ((9 * cel) + 160) / 5;
        System.out.println("A nova temperatura é: "+far+" F"); 
        
    }
}

File: CC/Codigos_Fonte_Relatorio_1/micro01/micro01.pas
program micro01;

var 
    cel, fars : real;

begin
    writeln('   Tabela de conversão: Celsius -> Fahrenheit');
    write('Digite a temperatura em Celsius: ');
    read(cel);

    fars := ((9*cel) + 160) / 5;

    writeln('A nova temperatura eh: ', fars, ' F'); 
end.
